onadata/apps/api/viewsets/project_viewset.py

```diff
@@ -120,9 +140,6 @@ def forms(self, request, **kwargs):
                 if str_to_bool(published_by_formbuilder):
                     MetaData.published_by_formbuilder(survey, 'True')
 
-                # clear project from cache
-                safe_delete(f'{PROJ_OWNER_CACHE}{survey.project.pk}')
```

Comment:
Removed this call, in favor of the changes introduced here

https://github.com/onaio/onadata/pull/1894/files#diff-3d65ce77835a2f07194046eb827e8fe1R420-R424